go/obscuronode/enclave/sql/edb_attestation.go

```diff
@@ -0,0 +1,215 @@
+package sql
+
+// An obscuro implementation of the ERA (Edgeless remote attestation) tool (which is basically just a small json schema
+// that Edgeless use as a standard data blob to encrypt into their attestation reports, includes signerID, security version etc.)
+
+// Initially forked from https://github.com/edgelesssys/era/blob/master/era/era.go but reworked to have an obscuro-friendly
+// 	 API that just included the things we were using.
+
+import (
+	"bytes"
+	"crypto/sha256"
+	"crypto/tls"
+	"encoding/binary"
+	"encoding/hex"
+	"encoding/json"
+	"encoding/pem"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+
+	"github.com/obscuronet/obscuro-playground/go/log"
+
+	"github.com/edgelesssys/ego/attestation"
+	"github.com/edgelesssys/ego/attestation/tcbstatus"
+	"github.com/edgelesssys/ego/enclave"
+	"github.com/tidwall/gjson"
+)
+
+const (
+	quoteEndpoint = "quote"
+	msgJSONField  = "message"
+	dataJSONField = "data"
+)
+
+type certQuoteResp struct {
+	Cert  string
+	Quote []byte
+}
+
+type EdgelessAttestationConstraints struct {
```

Comment:
Nice one.
This could use some comments. I noticed there is some logic around the UniqueId, and can't figure out the reason